jormungandr/src/blockchain/chain.rs

```diff
@@ -49,11 +49,13 @@ See Internal documentation for more details: doc/internal_design.md
 [`Branch`]: ./struct.Branch.html
 */
 
+use super::{branch::Branches, reference_cache::RefCache};
```

Comment:
why did these include moves?

Reply:
I removed re-exports from the top of `mod blockchain` that were only used internally in the module, and minimized exports in general. This allows to catch dead code lints in more places.